nerdlets/e2m-gui-nerdlet/account-details.js

```diff
@@ -116,7 +116,7 @@ export default class AccountDetails extends React.Component {
 
 AccountDetails.propTypes = {
   filteredMetrics: PropTypes.array,
-  accountIdFilter: PropTypes.string,
+  accountIdFilter: PropTypes.number,
```

Comment:
I think this may actually be a string right, its the "filter" and not an ID?

Reply:
its being used in this function, and best i can tell its trying to get a key from an object, which i'd assume would be a string? could be 💯 wrong if it was yelling at you tho....

```js
export function filterCardinalityTotalsByAccountIdFilter(
  cardinalityTotals,
  accountIdFilter
) {
  if (!cardinalityTotals || !accountIdFilter) {
    return cardinalityTotals;
  }
  const filteredTotal = {};
  filteredTotal[accountIdFilter] = cardinalityTotals[accountIdFilter];
  return filteredTotal;
}
```

Reply:
I was getting a propTypes error in the logs, which was the reason for this change. I was thinking the same, which made it difficult to locate. The component was receiving a number.